pkgs/top-level/all-packages.nix

```diff
@@ -27722,15 +27722,18 @@ with pkgs;
 
   usermount = callPackage ../os-specific/linux/usermount { };
 
-  util-linux = if stdenv.isLinux then callPackage ../os-specific/linux/util-linux { }
-              else unixtools.util-linux;
+  util-linux = (callPackage ../os-specific/linux/util-linux { }).override {
```

Comment:
```suggestion
  util-linux = callPackage ../os-specific/linux/util-linux {
    capabilitiesSupport = stdenv.isLinux;
    writeSupport = stdenv.isLinux;
    shadowSupport = stdenv.isLinux;
  };
```
Is probably the preferred style. Or maybe moving these defaults into the `default.nix` instead of in `all-packages.nix`? Not sure what the current best-practice is :thinking:

Reply:
Moved the settings to `default.nix`. Thanks.